Dockerfile

```
@@ -20,7 +20,7 @@ COPY eventservice/events.yaml /usr/share/nginx/html/app/
 COPY profileservice/profile.yaml /usr/share/nginx/html/app/
 COPY loggingservice/logging.yaml /usr/share/nginx/html/app/
 
-ENV URLS "[{url: 'app/appconfig.yaml', name: 'App Config Building Block'}, {url: 'app/auth.yaml', name: 'Authentication Building Block'}, {url: 'app/events.yaml', name: 'Events Building Block'}, {url: 'app/profile.yaml', name: 'Profile Building Block'}, {url: 'app/logging.yaml', name: 'Logging Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/health/doc', name: 'Health Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/talent-chooser/doc', name: 'Talent Chooser Building Block'} ]"
+ENV URLS "[{url: 'app/appconfig.yaml', name: 'App Config Building Block'}, {url: 'app/auth.yaml', name: 'Authentication Building Block'}, {url: 'app/events.yaml', name: 'Events Building Block'}, {url: 'app/profile.yaml', name: 'Profile Building Block'}, {url: 'app/logging.yaml', name: 'Logging Building Block'}, {url: 'https://api.rokwire.illinois.edu/health/doc', name: 'Health Building Block'}, {url: 'https://api.rokwire.illinois.edu/talent-chooser/doc', name: 'Talent Chooser Building Block'} ]"
```

```suggestion
ENV URLS "[{url: 'app/appconfig.yaml', name: 'App Config Building Block'}, {url: 'app/auth.yaml', name: 'Authentication Building Block'}, {url: 'app/events.yaml', name: 'Events Building Block'}, {url: 'app/profile.yaml', name: 'Profile Building Block'}, {url: 'app/logging.yaml', name: 'Logging Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/health/doc', name: 'Health Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/talent-chooser/doc', name: 'Talent Chooser Building Block'} ]"
```

Suggesting this change as probably the source branch was master.

---

changed